mmv1/templates/terraform/examples/firebasehosting_site_basic.tf.erb

```diff
@@ -0,0 +1,5 @@
+resource "google_firebase_hosting_site" "default" {
+  provider = google-beta
+  project  = "<%= ctx[:test_env_vars]['project_id'] %>"
+  site = "tf-test-site%{random_suffix}"
```

Comment:
This is currently showing up in the docs: https://github.com/modular-magician/terraform-provider-google/compare/auto-pr-6598-old..auto-pr-6598#diff-6346016a04a9b44616f4908e7c618c06ab6259c65c98d0e4d3fdc3c0d51958faR41

Make the changes in terraform.yaml to pass a var for site_id and then change the line here & in the full tests to:

```suggestion
  site = "<%= ctx[:vars]['site_id'] %>"
```